maps/dungeons/tiered_dungeon.lua

```diff
@@ -12,8 +12,17 @@ local BiterRaffle = require 'functions.biter_raffle'
 local Functions = require 'maps.dungeons.functions'
 local Get_noise = require 'utils.get_noise'
 local Alert = require 'utils.alert'
+local Research = require 'maps.dungeons.research'
 local DungeonsTable = require 'maps.dungeons.table'
+local BottomFrame = require 'comfy_panel.bottom_frame'
+local Autostash = require 'modules.autostash'
+local Panel = require 'comfy_panel.config'
+Panel.get('gui_config').spaghett.noop = true
+local Collapse = require 'modules.collapse'
```

Comment:
modules.rpg.main requires modules.explosives
modules.explosives requires modules.collapse

After about a week of running the server, collapse started trying to collapse tiered dungeons and by the time I noticed I had a 12G log file. So in 
https://github.com/eric-anderson/ComfyFactorio/commit/73b5d03276e8202f62689e7e39d5c934365e94c4

I added the ability to disable collapse entirely and in this commit I disabled collapse.

I didn't feel confident enough in understanding the tree of dependencies in the modules to make a change elsewhere.

Reply:
I took another look at it. It looks like if I call Collapse.start_now(false) it will also disable collapse.  It's slightly less safe than the other approach since if dungeons on_init runs before collapse on_init then it won't work right, but based on some manual testing it seems to do the right thing.  Hopefully it won't break after a week.